openaddresses/be/bru/bosa-be-nl.json

```diff
@@ -0,0 +1,38 @@
+{
+    "data": "FIXME insert correct url here",
+    "protocol": "http",
+    "coverage": {
+         "ISO 3166": {
+             "alpha2": "BE-BRU",
+             "state": "Brussels-Capital Region",
+             "country": "Belgium"
+         },
+         "country": "be",
+         "state": "brussels-capital region",
+         "geometry" :{ "type": "Polygon", "coordinates": [ [ [ 4.271, 50.837 ], [ 4.272, 50.839 ], [ 4.282, 50.838 ], [ 4.284, 50.847 ], [ 4.287, 50.849 ], [ 4.283, 50.854 ], [ 4.289, 50.856 ], [ 4.282, 50.861 ], [ 4.28, 50.865 ], [ 4.284, 50.871 ], [ 4.292, 50.873 ], [ 4.294, 50.876 ], [ 4.299, 50.879 ], [ 4.298, 50.885 ], [ 4.295, 50.885 ], [ 4.295, 50.887 ], [ 4.293, 50.887 ], [ 4.295, 50.89 ], [ 4.3, 50.89 ], [ 4.303, 50.892 ], [ 4.31, 50.891 ], [ 4.311, 50.893 ], [ 4.319, 50.895 ], [ 4.324, 50.899 ], [ 4.341, 50.903 ], [ 4.364, 50.901 ], [ 4.378, 50.896 ], [ 4.38, 50.901 ], [ 4.384, 50.902 ], [ 4.388, 50.911 ], [ 4.396, 50.91 ], [ 4.398, 50.913 ], [ 4.405, 50.914 ], [ 4.415, 50.911 ], [ 4.413, 50.908 ], [ 4.418, 50.906 ], [ 4.418, 50.904 ], [ 4.427, 50.901 ], [ 4.426, 50.899 ], [ 4.433, 50.894 ], [ 4.426, 50.891 ], [ 4.437, 50.879 ], [ 4.427, 50.878 ], [ 4.428, 50.876 ], [ 4.427, 50.872 ], [ 4.421, 50.868 ], [ 4.426, 50.863 ], [ 4.431, 50.863 ], [ 4.43, 50.861 ], [ 4.438, 50.858 ], [ 4.439, 50.861 ], [ 4.447, 50.857 ], [ 4.446, 50.855 ], [ 4.461, 50.852 ], [ 4.465, 50.844 ], [ 4.467, 50.844 ], [ 4.468, 50.836 ], [ 4.466, 50.836 ], [ 4.477, 50.82 ], [ 4.455, 50.819 ], [ 4.457, 50.817 ], [ 4.455, 50.815 ], [ 4.449, 50.812 ], [ 4.449, 50.808 ], [ 4.479, 50.793 ], [ 4.476, 50.793 ], [ 4.472, 50.79 ], [ 4.463, 50.789 ], [ 4.461, 50.786 ], [ 4.435, 50.777 ], [ 4.433, 50.779 ], [ 4.383, 50.764 ], [ 4.382, 50.766 ], [ 4.384, 50.766 ], [ 4.382, 50.778 ], [ 4.384, 50.779 ], [ 4.382, 50.778 ], [ 4.382, 50.782 ], [ 4.38, 50.782 ], [ 4.375, 50.789 ], [ 4.363, 50.785 ], [ 4.355, 50.792 ], [ 4.343, 50.786 ], [ 4.339, 50.787 ], [ 4.34, 50.789 ], [ 4.334, 50.789 ], [ 4.34, 50.784 ], [ 4.336, 50.78 ], [ 4.332, 50.78 ], [ 4.336, 50.778 ], [ 4.335, 50.777 ], [ 4.327, 50.781 ], [ 4.326, 50.78 ], [ 4.322, 50.783 ], [ 4.317, 50.796 ], [ 4.306, 50.798 ], [ 4.302, 50.804 ], [ 4.306, 50.813 ], [ 4.299, 50.812 ], [ 4.296, 50.809 ], [ 4.285, 50.808 ], [ 4.276, 50.809 ], [ 4.276, 50.812 ], [ 4.278, 50.813 ], [ 4.278, 50.816 ], [ 4.272, 50.815 ], [ 4.271, 50.813 ], [ 4.266, 50.812 ], [ 4.256, 50.812 ], [ 4.257, 50.816 ], [ 4.277, 50.82 ], [ 4.282, 50.82 ], [ 4.283, 50.824 ], [ 4.282, 50.826 ], [ 4.277, 50.825 ], [ 4.273, 50.828 ], [ 4.273, 50.832 ], [ 4.276, 50.833 ], [ 4.276, 50.836 ], [ 4.271, 50.837 ] ], [ [ 4.385, 50.79 ], [ 4.385, 50.79 ], [ 4.386, 50.79 ], [ 4.385, 50.79 ] ] ] }
+    },
+    "conform": {
+        "format": "csv",
+        "srs": "EPSG:4326",
+        "file": "brussels_addresses.csv",
+        "encoding": "utf-8",
+        "accuracy": "3",
+        "number": "house_number",
+        "street": "streetname_nl",
```

Comment:
IIRC it is also possible to add box number as "unit" ?

Reply:
Yes! I just added it!